web/ci.sh

```diff
@@ -42,6 +42,21 @@ builder_parse "$@"
 TIER=`cat ../TIER.md`
 BUILD_NUMBER=`cat ../VERSION.md`
 
+function web_sentry_upload () {
+  if [ $1 = "webview" ]; then
+    # There is no "publish" version for app/webview; it's "published" inside our mobile apps.
+    ARTIFACT_FOLDER="$KEYMAN_ROOT/web/build/app/webview/release/"
+  elif [ $1 = "browser" ]; then
+    ARTIFACT_FOLDER="$KEYMAN_ROOT/web/build/publish/release/"
+  fi
+
+  pushd "$ARTIFACT_FOLDER"
+  echo "Uploading to Sentry..."
+  npm run sentry-cli -- releases files "$SENTRY_RELEASE_VERSION" upload-sourcemaps --strip-common-prefix "@keymanapp/keyman/" --rewrite --ext js --ext map --ext ts || fail "Sentry upload failed."
+  echo "Upload successful."
+  popd
+}
```

Comment:
For comparison, from our current `stable-16.0` build script:

https://github.com/keymanapp/keyman/blob/7a67894bbbdfe53b0da1c85dfc5b3d9a1965c74d/web/source/build.sh#L553-L560

Reply:
Note that post-modularization, to ensure that pathing to all components (even outside `web/`) works correctly, the "sourcesRoot" is set to `@keymanapp/keyman/<repo path>` as of #8995.

Reply:
I've since nixed the `npm run` and `--` parts of the `sentry-cli` command.  I'd forgotten that it's not a npm script anymore and that we can run it directly now.